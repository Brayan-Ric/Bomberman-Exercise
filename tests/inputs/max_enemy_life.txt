W B1
F4 _