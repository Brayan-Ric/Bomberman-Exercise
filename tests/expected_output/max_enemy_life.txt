Error: La vida de un enemigo esta fuera del rango. Rango: 1 al 3